app/controllers/sufia/admin/admin_sets_controller.rb

```diff
@@ -29,6 +29,7 @@ def index
       add_breadcrumb t(:'sufia.toolbar.admin.menu'), sufia.admin_path
       add_breadcrumb t(:'sufia.admin.sidebar.admin_sets'), sufia.admin_admin_sets_path
       @admin_sets = CurationConcerns::AdminSetService.new(self).search_results(:read)
+      @presenter = presenter_class.new(SolrDocument, current_ability)
```

Comment:
@andjsmit I now realize the ticket you're working on could be clearer -- there are *two* bits of AdminSet work in Sufia at the moment: the stuff in the admin dashboard and the stuff outside the admin dashboard. I believe the ticket you're working on is for the stuff *not* in the admin dashboard, so you should be using the AdminSetsController not under `sufia/admin/`, if that makes sense.